Nuro/Utilities/CoreData/Repository/CategoryLocalRepository.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  CategoryRepository.swift
+//  Nuro
+//
+//  Created by Karen Natalia on 11/10/22.
+//
+
+import Foundation
+import CoreData
+
+protocol CategoryRepository {
+    func getAll() -> [Category]
+    func addCategory(name: String)
+}
```

Comment:
_Discussion_: Why not extract all CoreData protocols to another file?
ex: `CoreDataProtocols`